commands/alias.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable no-nested-ternary */
+const ora = require('ora');
+const { configStore, getDefaults } = require('@eik/common-config-loader');
+const { getCWD, logger } = require('../utils');
+
+const Alias = require('../classes/alias');
+const { Alias: AliasFormatter } = require('../formatters');
+
+exports.command = 'alias <name> <version> <alias>';
+
+exports.aliases = [
+    'ma',
+    'pa',
+    'pkg-alias',
+    'na',
+    'dep-alias',
+    'dependency-alias',
+];
+
+exports.describe = `Create a semver major alias for a package or an import map as identified by its name and version.
+    A package or import map with the given name and version must already exist on asset server
+    Alias should be the semver major part of the import map version.
+    Eg. For a package or import map of version 5.4.3, you should use 5 as the alias`;
+
+exports.builder = (yargs) => {
+    const cwd = getCWD();
+    const defaults = getDefaults(cwd);
+
+    yargs
+        .positional('name', {
+            describe: `Name of package or map that is to be aliased`,
+            type: 'string',
+        })
+        .positional('version', {
+            describe: `Version for package or map that is to be aliased`,
+            type: 'string',
+        })
+        .positional('alias', {
+            describe: `Alias for a semver version. Should be the semver major component of version.`,
+            type: 'string',
+        });
+
+    yargs.options({
+        server: {
+            alias: 's',
+            describe: 'Specify location of asset server.',
+            default: defaults.server,
+        },
+        cwd: {
+            alias: 'c',
+            describe: 'Alter current working directory.',
+            default: defaults.cwd,
+        },
+        debug: {
+            describe: 'Logs additional messages',
```

Comment:
```suggestion
            describe: 'Log additional messages',
```

For all of these we should be consistent with tense. Don't mind which but either Logs additional messages, Alters current working directory etc or Log additional messages, alter current working directory etc.